recursion/problem-set-submit.checkpoint.md

```diff
@@ -0,0 +1,43 @@
+# Problem Set Submission
+
+Instructors provide code feedback through pull requests.
+
+In order to get code feedback on [Problem Set: Recursion Pt. 1](problem-set-recursion-pt-1.md) and [Problem Set: Recursion Pt. 2](problem-set-recursion-pt-2.md), follow these directions:
+
+1. Fork [this repo, `core-problem-set-recursion`](https://github.com/AdaGold/core-problem-set-recursion) to your GitHub account
```

Comment:
Is it intentionally to have them fork directly off of AdaGold? I'm not opposed to this, and it will help Accelerate (since right now we would be planning to run accelerate's interlude from main). If this gets changed to Ada-C15, we'd need to branch interlude for accelerate, as we have done for core-projects.

Reply:
Just to repeat what I mentioned on Slack--

Let's play it out and fork off AdaGold. I think there's pros and cons to this, but for now, let's try it. I'd like to see it break on this assignment, if anything :P